.github/workflows/vercel.yml

```diff
@@ -19,6 +19,8 @@ jobs:
 
     steps:
       - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.head_ref || github.ref_name }}
```

Comment:
This change makes it so the branch name and commit message are displayed for [Vercel deployments](https://vercel.com/terraformation/tree-location-web/deployments) rather than `HEAD` & `Merge [SHA]`.